PiwikTracker/PiwikUserDefaults.swift

```diff
@@ -5,8 +5,11 @@ import Foundation
 /// All getter and setter are sideeffect free and automatically syncronize
 /// after writing.
 internal struct PiwikUserDefaults {
-    static var standard = PiwikUserDefaults()
-    let userDefaults = UserDefaults.standard
+    let userDefaults: UserDefaults
+
+    init(suiteName: String?) {
```

Comment:
I guess for some (advanced) integration tests your need to control the starting values of some of the settings. It might be worth using an interface.

````
protocol SiteSettings {
   var totalNumberOfVisits: Int { get set }
}

extension UserDefaults: SiteSettings {

    func siteSettings(key: String) -> SiteSettings? {
          return UserDefaults(suiteName: key)
   }

    var totalNumberOfVisits: Int {
        get {
            //
        }
        set {
            //
        }
    }
}
````